### ISSUE TITLE
Can't boot from USB stick after release 34350.

### ISSUE THREAD

**Issue by lkmishra** [clearlinux/distribution#2310]
I use clear linux from an USB stick. It was working fine till release 34350. After updating to 34360, it is not booting any more. I repair it from another bootable usb. But it is not booting anymore. I tried from 34370 and 34380. No difference. It's still not booting. I also tried with another stick installed with clear linux, no difference.

**Comment by paulcarroty** [comment]
Can you provide some logs?

> I use clear linux from an USB stick

Please don't, USB is very slow on operations with a ton of small files and die fast.

**Comment by lkmishra** [comment]
> Can you provide some logs?
> 
> > I use clear linux from an USB stick
> 
> Please don't, USB is very slow on operations with a ton of small files and die fast.

It is still much faster than windows in USB. I will use ssd after buying another PC.
Here is some log files I got from /var/log directory, not sure which one you want. But it's not booting at all from USB now, directly booting form hard disk. Not sure it would generate any logs.
[Xorg.0.log](https://github.com/clearlinux/distribution/files/6168297/Xorg.0.log)

I tried to upload lastlog & wtmp file. but it is saying not supported.

**Comment by lkmishra** [comment]
It was working for 34420 and 34430. But the problem started again on 34440. Can't boot now, again.

While it was working on 34420, I noticed, if I do write operation on another USB, the problem started again. It worked again after repairing.

Also it doesn't detect the USB stick sometime from installation disk. Needed to reinsert the USB after booting.

**Comment by lebensterben** [comment]
Your peculiar use case is not likely to be well supported.

See https://docs.01.org/clearlinux/latest/about.html?highlight=gnome#id5

**Comment by lkmishra** [comment]
> Your peculiar use case is not likely to be well supported.
> 
> See https://docs.01.org/clearlinux/latest/about.html#id5

I don't think I am the only person using Clear linux from memory stick. There are many more. You can see in the forum messages.

**Comment by 0xA1B2** [comment]
Agree, I also used a USB stick for many months, just swap performance was horrible. A cheap SSD is for sure a better choice if you can afford it.

**Comment by lkmishra** [comment]
It is still causing the problem. Most of the time I have to repair the memory stick to boot. It's boot on it's own very very rarely. Almost everyday I have to repair the stick for booting. It seems it's not flushing the stick before shutting down.
  I can boot from the installation stick every time without any problem.
I am using samsung memory stick fast speed in USB3 port.
